README.md

```diff
@@ -98,8 +134,8 @@ Leiningen/Boot:
 
 ## License
 
-Distributed under the [MIT License].  
-Copyright (c) 2023 [Scarlet Global Holdings Ltd][scarlet]
+Distributed under the [MIT License].
```

Comment:
The trailing spaces were intentional -- this is [Markdown line break](https://www.markdownguide.org/basic-syntax/#line-breaks)